app/src/main/java/fr/free/nrw/commons/auth/SignupActivity.java

```diff
@@ -61,4 +63,13 @@ public void onBackPressed() {
             super.onBackPressed();
         }
     }
+
```

Comment:
Would you mind adding javadoc here explaining why this code is necessary, and linking to https://github.com/commons-app/apps-android-commons/issues/5303? Thanks :-)

Reply:
added

Reply:
Great doc, very useful to future readers, thanks! :-)